torch/testing/_internal/distributed/rpc/rpc_test.py

```diff
@@ -1107,7 +1108,11 @@ def test_return_local_rrefs(self):
             for rref in rref_list
         ]
 
+        info = _rref_context_get_debug_info()
         self.assertEqual(rets, [11, 12, 13])
+        print(info["num_forks"])
```

Comment:
Remove this print or add more details like print("Num RRef forks: {}".format(num_forks))